tox.ini

```diff
@@ -0,0 +1,39 @@
+[tox]
+minversion = 3.1
+envlist = py27,docs,style
```

Comment:
`py27` feels like the wrong default to me...

Should this be `Py37` or `Py38`?

Reply:
Whoops! Yes, indeed. Corrected.